cli/console.py

```diff
@@ -204,5 +203,65 @@ def modify_task(
         raise e
 
 
+@app.command("bulk-update")
+def bulk_update(
+    job_id: Annotated[
+        str, typer.Option("--job-id", "-j", help="Job ID to update tasks for")
+    ],
+):
+    console.print(
+        f"[bold red] :loudspeaker: This command will modify all tasks for {job_id}. Proceed with caution."
+    )
+    sp_credential = obtain_sp_credential()
+    container_name = azure_storage["azure_container_name"]
+    try:
+        bulk_updates = update_config_bulk(sample_task, modifiable_params, console)
+        if bulk_updates:
+            # If there are updates, download all of the existing tasks for the job,
+            # update the fields, and generate new tasks with updated timestamps
+            with console.status(
+                ":cloud: Task configuration updated successfully, pushing to Azure...\n",
+                spinner="aesthetic",
+            ):
+                blob_service_client = instantiate_blob_service_client(
+                    sp_credential=sp_credential,
+                    account_url=azure_storage["azure_storage_account_url"],
+                )
+                container_client = blob_service_client.get_container_client(
+                    container_name
+                )
+                blob_list = container_client.list_blobs()
+                tasks_for_job = get_tasks_for_job_id(blob_list=blob_list, job_id=job_id)
+                for task_filename in tasks_for_job:
+                    full_blob_path = f"{job_id}/{task_filename}"
+                    blob_data = download_blob(
+                        blob_path=full_blob_path, sp_credential=sp_credential
+                    )
+                    updated_config = dict(blob_data, **bulk_updates)
+
+                    # Update timestamps and push to Azure
+                    timestamp = generate_timestamp()
+                    updated_config["as_of_date"] = timestamp
```

Comment:
So we don't have `as_of_date` in the original? This is just for usefulness down the road?

Reply:
Oh shoot, I forgot that we moved `as_of_date` to the `parameters` -- maybe I can remove this? Or change this to `date_modified` or something else, what do you think?

Reply:
It gets generated initially here https://github.com/CDCgov/cfa-config-generator/blob/main/pipelines/epinow2/generate_config.py#L34

Reply:
I would maybe move the assignment into `parameters`